cli.js

```diff
@@ -51,9 +51,25 @@ async function applyCommand() {
   await applyScenario(scenario);
 }
 
+const configExtensions = [".js", ".mjs"];
```

Comment:
.js === module when type=module, like in my demo repo -- even tho I kept mjs for now